go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/fatedier/frp
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.4
```

Comment:
Can we skip adding this?

Reply:
I tried, but the linter failed claiming that go mod tidy should run.
You can try removing that with go 1.22 and running go mod tidy to see what happens, it might provide better results.

Reply:
I did not encounter any issues in my local go1.23.3 environment.

Reply:
See now, without that mod tidy

Reply:
Strange problem, it seems that `go 1.22.0` needs to be retained.

Reply:
Done